.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - pwd
   - chmod +x ./scripts/ci/install_myhdl.sh
   - ./scripts/ci/install_myhdl.sh
+  - pip install -e git://github.com/cfelton/rhea.git#egg=rhea
```

Comment:
I think the actual script we want, is to use

>> pip install . -r requirements.txt 

below and remove the install_myhdl script.  Then our virtualenv installs should work the same as the travis installs.  We can change this later but I wanted to point it out since there are changes in the PR.